index.html

```diff
@@ -10,34 +10,33 @@
     <script src="utils.js" defer></script>
     <script type="module" src="script.js" defer></script>
   </head>
+
   <body>
     <nav id="tasksNav">
       <a id="createTask" class="nav-link"> Tasks </a>
     </nav>
 
     <section class="buttonSection">
-      <button
-        class="create-task-btn"
-        type="button"
-        onclick="location.href='task/index.html'"
-      >
+      <a class="create-task-btn link-button" href="task/index.html">
         Create Tasks
-      </button>
-      <button
-        class="profile-task-btn"
-        type="button"
-        onclick="location.href='profile/index.html'"
-      >
+      </a>
+      <a class="profile-task-btn link-button" href="goal/index.html">
+        Create Goals
+      </a>
+      <a class="profile-task-btn link-button" href="profile/index.html">
         Profile
-      </button>
+      </a>
       <a
         id="user-management-link"
-        class="element-display-remove"
+        class="element-display-remove link-button"
         href="user-management/index.html"
       >
         User Management
       </a>
-      <a href="online-members/online-members.html" class="online-members-link">
+      <a
+        href="online-members/online-members.html"
+        class="online-members-link link-button"
+      >
```

Comment:
I think you should make these changes in separate PR, Things that are not related to your PR are to be done in a new PR. Please raise a new PR for this

Reply:
done